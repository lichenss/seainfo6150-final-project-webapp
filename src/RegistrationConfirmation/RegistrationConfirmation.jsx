import React from 'react'
import Footer from '../Footer/Footer'
import Header from '../Header/Header'

const RegistrationConfirmation = () => {
    return (
        <div>
            <Header />
            
            <Footer />
        </div>
    )
}
export default RegistrationConfirmation