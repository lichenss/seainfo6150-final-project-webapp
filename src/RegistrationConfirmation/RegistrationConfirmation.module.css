.section{
  background-color: rgb(229, 231, 233);
}
.container {
    margin-top: 70px;
    display: flex;
    flex-direction: column;
    justify-content: flex-start;
    align-items: center;
  }
  .title{
    font-family:'Roboto', sans-serif;
  }
  .content{
    font-family: 'Open Sans', sans-serif;
  }