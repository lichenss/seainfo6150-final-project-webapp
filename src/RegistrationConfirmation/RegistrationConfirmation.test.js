import React from "react";
import RegistrationConfirmation from "./RegistrationConfirmation";
import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";

var data={
    title:"Therapeutic Art Life Coach Certification (Accredited)",
    imgurl:"https://arhu.umd.edu/sites/default/files/2019-04/iStock-955549248%20composite-JC%20flat2%20opt1-min.jpg",
    description:"Therapeutic Art: Help Clients Heal, Reduce Stress & Manifest Dreams with Mandalas, Affirmations, Vision Boards",
    instructor:"Janet Giessl"
};
const location={
        state:{data}
}
describe("RegistrationConfirmation tests", () => {
    it("renders correctly", () => {
      const { container } = render(
      <BrowserRouter>
      <RegistrationConfirmation loaction={location} />
      </BrowserRouter>);
      expect(container).toMatchSnapshot();
    });
  });