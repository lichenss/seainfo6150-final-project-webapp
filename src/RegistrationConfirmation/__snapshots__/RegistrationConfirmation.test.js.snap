// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationConfirmation tests renders correctly 1`] = `
<div>
  <div>
    <header
      class="header"
    >
      <div
        class="navbar"
      >
        <button
          class="logo"
        />
        <div>
          <a
            class="catLink"
            href="/category/computer-science"
          >
            Computer Science
          </a>
          <a
            class="catLink"
            href="/category/business"
          >
            Business
          </a>
          <a
            class="catLink"
            href="/category/data-science"
          >
            Data Science
          </a>
          <a
            class="catLink"
            href="/category/arts-and-humanities"
          >
            Arts and Humanities
          </a>
          <a
            class="catLink"
            href="/category/physical-science-and-engineering"
          >
            Physical Science and Engineering
          </a>
          <a
            class="catLink"
            href="/category/math-and-logic"
          >
            Math and Logic
          </a>
        </div>
        <div
          class="links"
        >
          <a
            class="barLink"
            href="/log-in"
          >
            Log In
          </a>
          <a
            class="barLink"
            href="/sign-up"
          >
            Sign Up
          </a>
        </div>
      </div>
    </header>
  </div>
  <section
    class="section"
  >
    <div
      class="container"
    >
      <div
        class="titleContainer"
      >
        <h2
          class="title"
        >
          Registration Confirmation
        </h2>
      </div>
      <div
        class="titleContainer"
      >
        <h3
          class="title"
        >
          Course Name:
        </h3>
      </div>
      <p
        class="content"
      >
        Therapeutic Art Life Coach Certification (Accredited)
      </p>
      <div
        class="instrcutorContainer"
      >
        <h3
          class="title"
        >
          Instructor:
        </h3>
        <p
          class="content"
        >
          Janet Giessl
        </p>
      </div>
      <div>
        <button
          class="button"
        >
           Cancel
        </button>
        <button
          class="button"
        >
           Confirm
        </button>
      </div>
    </div>
  </section>
  <footer
    class="footer"
  >
    <div
      class="links"
    >
      <a
        class="link"
        href="/contact-us"
      >
        <h3>
          Contact Us
        </h3>
      </a>
      <a
        class="link"
        href="/about-us"
      >
        <h3>
          About Us
        </h3>
      </a>
    </div>
  </footer>
</div>
`;
